### PR TITLE
fix sending announcement-mails

### PR DESCRIPTION
**Pull request by cherti**
This fixes termine's ability to send out announcements for c1/4 and Stammtisch.

### CODE REVIEW

File: termine/announce.go [hunk 1]
```diff
@@ -115,7 +116,7 @@ func sendAnnouncement(subject string, msg []byte) error {
 		return err
 	}
 
-	cmd := exec.Command("/bin/true", "-t")
+	cmd := exec.Command("/usr/sbin/sendmail", "-t")
```
Comment (Merovius) [inline termine/announce.go]: lol

File: termine/announce.go [hunk 2]
```diff
@@ -42,10 +42,11 @@ func init() {
 func announceStammtisch(t *data.Termin) error {
 	maildraft := `Liebe Treffler,
 
-am kommenden Donnerstag ist wieder Stammtisch. Diesmal sind wir bei {{.Location}}.
+am kommenden Donnerstag ist wieder Stammtisch. Diesmal sind wir bei
+{{.Location}}.
```
Comment (cherti) [inline termine/announce.go]: The interesting line we briefly talked about is [this one](https://github.com/nnev/website/pull/20/files?diff=unified#diff-0039b705f632a6bd78af8bb434d2b8e7R110). According to [this example](https://play.golang.org/p/mSLxd0G1Ag), it should work fine placing `{{.Location}}.` right after the `sind wir bei`.
However, doing so results in the message text being cut after the inserted location in the mail sent. The `.` after the `{{.Location}}` is also dropped, along with the rest of the message.
The `msg` that the `body.Writer(msg)` gets is still the full text. However, it is not written entirely for some reason. [According to the godoc, as far as I understand it, the Writer-interface is not allowed to drop text without returning an error.](https://godoc.org/io#Writer) However, here it seems to be doing that.

Any obvious explanation for that behavior? any obvious bug I have overseen?

This type of text addresses the issue and can be merged without problems, but I find this behavior of the code… rather unsatisfying, as long as I haven't understood what's going wrong here.

File: termine/announce.go (hunk 1 above)
Comment (cherti) [inline termine/announce.go]: yep^^

File: termine/announce.go (hunk 2 above)
Comment (cherti) [inline termine/announce.go]: [solution of the problem for historical documentation](https://github.com/nnev/website/commit/d73c95a128cf55c9d25166f0c20e7c001e8bec93)

**Pull request merged**